### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This is an umbrella PR fixing minor issues listed in https://trello.com/c/9LpdBtMl/10-web-issues

- encoding fixes, for example `beta'Cop` http://idr-ci.openmicroscopy.org:1880/web/webclient/?show=gene-beta'Cop
- metdata pane / details tab always open

This PR requires https://github.com/openmicroscopy/openmicroscopy/pull/5014

### CODE REVIEW

**Comment by aleksandra-tarkowska** [comment]
THis is passing agains m52

```
+ omego download server --branch OMERO-build --ci https://10.0.51.135:8443
...
+ sed -i '' 's/^\(omero\.host\s*=\s*\).*$/\110.0.51.135/' /Users/ola/OMERO/DEV/metadata52/src/webapp.git/mapr.git/../OMERO.server/etc/ice.config
+ sed -i '' 's/^\(omero\.port\s*=\s*\).*$/\114064/' /Users/ola/OMERO/DEV/metadata52/src/webapp.git/mapr.git/../OMERO.server/etc/ice.config
...
============================================================ test session starts =============================================================
platform darwin -- Python 2.7.10, pytest-3.0.5, py-1.4.32, pluggy-0.4.0 -- /Users/ola/OMERO/DEV/metadata52/src/webapp.git/omerovenv/bin/python
cachedir: .cache
rootdir: /Users/ola/OMERO/DEV/metadata52/src/webapp.git/mapr.git, inifile: setup.cfg
plugins: django-3.1.2, logging-2015.11.4
collected 16 items 

tests/integration/test_config.py::TestMapr::test_settings PASSED
tests/integration/test_config.py::TestMapr::test_empty_settings XPASS
tests/integration/test_config.py::TestMapr::test_bad_settings[foo] PASSED
tests/integration/test_config.py::TestMapr::test_bad_settings[] PASSED
tests/integration/test_config.py::TestMapr::test_bad_settings[mygenes] PASSED
tests/integration/test_mapr.py::TestMapr::test_autocomplete[ac0] PASSED
tests/integration/test_mapr.py::TestMapr::test_autocomplete[ac1] PASSED
tests/integration/test_mapr.py::TestMapr::test_autocomplete[ac2] PASSED
tests/integration/test_mapr.py::TestMapr::test_autocomplete[ac3] PASSED
tests/unittests/test_version.py::TestUtil::test_version[version0] PASSED
tests/unittests/test_version.py::TestUtil::test_version[version1] PASSED
tests/unittests/test_version.py::TestUtil::test_version[version2] PASSED
tests/unittests/test_version.py::TestUtil::test_version[version3] PASSED
tests/unittests/test_version.py::TestUtil::test_version[version4] PASSED
tests/unittests/test_version.py::TestUtil::test_version[version5] PASSED
tests/unittests/test_version.py::TestUtil::test_version[version6] PASSED

=================================================== 15 passed, 1 xpassed in 34.38 seconds ====================================================
```

Looks like populate metadata plug-in in main line doesn't like particular characters. I can remove test, but the rest is needed for a release

**Comment by aleksandra-tarkowska** [comment]
tests passed in https://10.0.51.135:8443/job/MAPR-test/23/console

**Comment by aleksandra-tarkowska** [comment]
I had to disable tests in travis, as metadata plugin in develop is not enough

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +23,8 @@ before_script:
   - python setup.py sdist install
 
 script:
-  - docker-compose -f docker/docker-compose.yml up --build --abort-on-container-exit
-  - exitcode=$(docker inspect -f '{{.State.ExitCode}}' docker_maprtest_1)
-  - docker-compose -f docker/docker-compose.yml rm -fv
-  - exit $exitcode
\ No newline at end of file
+  - echo 'Tested in CI'
```
Comment (rleigh-codelibre) [inline .travis.yml]: If it's tested in CI, and no longer using travis, then why not delete .travis.yml entirely and disable travis for this repo?

Comment (joshmoore) [inline .travis.yml]: Minimally, e.g., code style is still tested.

**Comment by will-moore** [comment]
Code changes look reasonable.
Let me know if you want me to test the UI (deployed somewhere)?

File: .travis.yml (hunk 1 above)
Comment (aleksandra-tarkowska) [inline .travis.yml]: That is temporary change, I would like travis to test it against build from both CIs.
In addition to code style it is testing pip installer

**Comment by aleksandra-tarkowska** [comment]
@will-moore link to web is in standup

**Comment by joshmoore** [comment]
Thanks guys. I need to tag 0.1.9 so I'm going to go ahead and merge. No objections to a 0.1.10 (and or 0.2.0) if need be.

**Pull request merged**